api/types/types.go

```diff
@@ -120,6 +120,9 @@ type Container struct {
 	SizeRootFs int               `json:",omitempty"`
 	Labels     map[string]string `json:",omitempty"`
 	Status     string            `json:",omitempty"`
+	HostConfig struct {
+		ContainerHostConfig string `json:"omitempty"`
```

Comment:
It also needs a `,` before `omitempty`. Otherwise go will assume that `omitempty` is the name of the json key.